test/workers/delete_account_hierarchy_worker_test.rb

```diff
@@ -50,4 +51,47 @@ def setup
 
     assert_raises(ActiveRecord::RecordNotFound) { provider.reload }
   end
+
+  class PerformTest < DeleteAccountHierarchyWorkerTest
```

Comment:
this is repeating about 99% of the previous test :joy: but i will help u out later :+1: i can't right now :disappointed:

Reply:
haven't seen that test.. fixed :)